frontstage/common/eq_payload.py

```diff
@@ -0,0 +1,120 @@
+import logging
+import time
+import uuid
+
+import iso8601
+from flask import current_app
+from frontstage.controllers import (collection_exercise_controller, collection_instrument_controller,
```

Comment:
Can we fix this import order please? In app imports shouldn't be with flask imports